org.eclipse.xtext/src/org/eclipse/xtext/xtext/XtextConfigurableIssueCodes.java

```diff
@@ -67,6 +68,8 @@ protected void initialize(IAcceptor<PreferenceKey> acceptor) {
 		acceptor.accept(create(BIDIRECTIONAL_REFERENCE, SeverityConverter.SEVERITY_WARNING));
 		acceptor.accept(create(DISCOURAGED_RULE_NAME, SeverityConverter.SEVERITY_WARNING));
 		acceptor.accept(create(USAGE_OF_DEPRECATED_RULE, SeverityConverter.SEVERITY_WARNING));
+		
```

Comment:
nitpick: why empty line here?

Reply:
I assumed the file was organised in semantic blocks and figured, that my new error code did not fit into the block above...
Would you prefer, to drop that empty line?